### ISSUE TITLE
”No results found!” on Windows 11

### ISSUE THREAD

**Issue by fakedeltatime** [pystardust/ani-cli#1240]
**Metadata (please complete the following information)**
Version: 4.60
OS: Windows 11
Shell: git bash inside Windows Preview Terminal
Anime: any, e.g. flcl

**Describe the bug**
Searching or trying to download anime errors out with the message ”No results found!”

**Steps To Reproduce**
1. Run `ani-cli -d flcl`

**Expected behavior**
Searching anime should work

**Comment by port19x** [comment]
Possible related to #1228

**Comment by Acrossy** [comment]
> Maybe it is blocked by countries or IPs, try switching to another IP. One of my VPN servers in Hong Kong was blocked a few months ago.

I tried using ProtonVPN to test that out since they have free servers but sadly the same problem persists on both a Netherlands and Poland servers. 
I can confirm that I was connected to both since the public ip shown were routing to those 2 countries
![2023-11-27 - 8](https://github.com/pystardust/ani-cli/assets/55287958/ebf52315-24b3-45b4-b930-bb20343d4b25)
![2023-11-27 - 9](https://github.com/pystardust/ani-cli/assets/55287958/7f865085-91ae-4651-96a9-b596a22ed274)

**Comment by Acrossy** [comment]
> I just downloaded ProtonVPN and connected to the free Netherlands server (169.150.218.59), and `ani-cli` works as expected. I also tried JP and US, and both work as well.

Damn it, I don't know what I'm doing wrong then, or if I am missing something :( 

Oh well... thank you for trying to help. If I manage to find something out I'll share but I'm honestly out of ideas to try.

No AVs, No Firewalls (the only AV/FW I use are the windows defender and the TP-link router's Firewalls), VPNs... nothing worked for me so far, so, i don't know what to try

**Comment by CoolnsX** [comment]
basically you have to remove somethings from the script

**Comment by Acrossy** [comment]
> basically you have to remove some lines of code from the script
> 
> Edit: this can be a potential fix, as I tested it with searching and it returned results

If you give me the details and tell me which specific file to edit I am willing to also try it out :)

**Comment by CoolnsX** [comment]
https://oshi.at/mGur/ani-cli

I modified and removed those code-blocks, can you test it?

just run this below code in your git-bash for downloading
```sh
curl -s "https://oshi.at/mGur/ani-cli" -o ani-cli && chmod +x ani-cli
```

and check it like this --
```sh
./ani-cli
```

**Comment by Acrossy** [comment]
> https://oshi.at/mGur/ani-cli
> 
> I modified and removed those code-blocks, can you test it?
> 
> just run this below code in your git-bash for downloading
> 
> ```shell
> curl -s "https://oshi.at/mGur/ani-cli" -o ani-cli && chmod +x ani-cli
> ```
> 
> and check it like this --
> 
> ```shell
> ./ani-cli
> ```

This worked! Takes a little while like a few seconds to fetch the info and the links but it does work! Can also confirm that going by "ani-cli" does not work, it has to be specifically ./ani-cli or ./ani-cli name-of-anime-here like you advised :) 
Thank you!

Out of curiosity tho I have the ani-cli file opened in VSCode and was really curious as to what was causing the issue? 
What did you remove exactly? 
Was it something related to the inputs not being recognized? 
Was it something about the way the queries were being processed? 
I'm genuinely curious :x

![2023-11-27 - 10](https://github.com/pystardust/ani-cli/assets/55287958/6e4f262f-ff5b-4d89-870c-a81820baa651)

**Comment by CoolnsX** [comment]
removed the ciphers arg from curl command, basically curl in git bash has limited scope and doesn't have ciphers like in linux or any unix-based operating system. And as I was testing in the morning, causes many people had this same issue, so I had to enable curl debug, which showed that it doesn't support any user specified ciphers.

cipher arg is one of the strategic disguising techniques to let the website know that request is coming from real browser. this helps the script from not getting blocked.

**Comment by CoolnsX** [comment]
now you have to install it to the path, so that you can able to use it like ani-cli instead of ./ani-cli

did you installed from scoop, right?

**Comment by CoolnsX** [comment]
the link fetching is windows-only issue, it's not your internet problem, it's a long story. it takes the same amount on mine, even though I have half-gigabit internet

**Comment by CoolnsX** [comment]
nah, it won't be a problem, as I said " it is one of the techniques", removing it won't be a problem and need not to worry about

**Comment by Acrossy** [comment]
> nah, it won't be a problem, as I said " it is one of the techniques", removing it won't be a problem and need not to worry about

Alright then!
Once again thank you so much for your help regarding this issue! :)

**Comment by idlukakas** [comment]
Should be removed it if the current os it's windows: --cipher "AES256-SHA256"

**Comment by ArekkusuMagni** [comment]
> now you have to install it to the path, so that you can able to use it like ani-cli instead of ./ani-cli
> 
> did you installed it from scoop, right?

So what is the command for installing it(./ani-cli) to Path on git bash?

**Comment by Acrossy** [comment]
> > now you have to install it to the path, so that you can able to use it like ani-cli instead of ./ani-cli
> > did you installed it from scoop, right?
> 
> So what is the command for installing it(./ani-cli) to Path on git bash?

I think there IS a way to copy/paste the file from "current directory" to another one. 
But if you mean the edited version of the script, in his case towards me, I went with "copying the file he gave me", which was an edited version of the "ani-cli" file without the cipher args and then paste it onto the directory where the original was "installed".

Note that this will get "undone" after you update the script through the command line. 
At least I noticed that on v .7 that the cipher args were back and I had to remove them my self.
